### PR TITLE
What clients we're looking for?

### PR DESCRIPTION
**Pull request by stlk**

### CODE REVIEW

File: client-evaluation/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+#### Are we able to maintain the project after it's finished
+
+We want to do that for limited time only. Amount of maintained projects is limited. Exception might be creating own product.
+
+#### Is the client willing to take over the project maintenance/development
+
+We're looking for clients willing to incorporate inhouse team. We want to talk with bussiness people.
+
+#### Does the client understand iterative development
```
Comment (f4z4on) [inline client-evaluation/README.md]: Iterative, evolutionary or continuous? Iterative presumes iterations which are only one tool to get from nothing to something… 😉

**Review by f4z4on — COMMENTED** [review]
Word picking… Ignore it if you don’t want to engage in etymological or philosophical discussion.

**Comment by f4z4on** [comment]
Although I agree with what you’ve written, I also think this is very broad and skewed towards one view of the issue… We’ve discussed many points – specifically things about specializations were a big topic. Perhaps notes from @mnrk move this PR somewhere else.

I’d also like to point out that written what to do from now on is only one part of successful change… Probably the most important one, but it needs more to be clear. I personally find what Michal Nygard described in [this article](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions "Documenting Architecture Decisions") a good starting point. It talks about documentation of architectural decisions through describing context, decision and consequences. I think not only architectural changes require at least that in one way or another, otherwise it gets lost why they became necessary and you’re destined to reinvent the wheel.

**Comment by f4z4on** [comment]
By the way, I realize I can write something to tackle the issues I mentioned above, but I think writing anything will be a waste of time at this point because @mnrk has notes. So, I’ll wait how this evolves…

**Comment by f4z4on** [comment]
Just a first comment after reading and thinking the text for a moment. This feels better… I think I see some far fetched logical conclusions but these insignificant. I’m not sure I agree with the way you describe software architure as an approach to make projects smaller. But if that were an example of one way how to achieve, I’d be fine.

File: client-evaluation/README.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+* **Critical importance of maintenance** is a sign that the project is very important for the client, but the client should know that it is effective to delegate repetitive and mundane tasks to hired employees or cheaper providers. Our team cannot internally handle expanding maintenance models, so creating automation and using services should be always part of the project.
 
-We're looking for clients willing to incorporate inhouse team. We want to talk with bussiness people.
+## Managing risk of the client
 
-#### Does the client understand iterative development
+Any software project is a risk and there is a chance that the project doesn't end up to the client satisfaction. There are two possible ways how to acceptably process such situation:
 
-#### Does the client accepts user testing
+1. Return money to the client - Make sure that the risk is included in the price and that we **can leave the project at any time** without additional costs for the client.
```
Comment (stlk) [inline client-evaluation/README.md]: I can't imagine any project we would be comfortable with money-back guarantee. This type of guarantee is suitable for products like books or our screen recording.

Comment (mnrk) [reply client-evaluation/README.md]: After a bit of thinking about it, that's the reason why I didn't call it "money back guarantee", but rather "Return money to the client" (upfront payment) or "Don't charge for our work and stop cooperation" (payment after acceptation).

Comment (mnrk) [reply client-evaluation/README.md]: I should probably change the text accordingly.

**Comment by mnrk** [comment]
@frzng What I meant was that there are actualy **only** two ways how to make a "project" smaller. First, lets define project = further unseparable continous work with a business outcome, second, there are these ways:

1. Changing the business outcome (cutting features, redefining product, etc)
2. Building individual technical blocks that obviously (for the client) lead to the business outcome. It is important that the client gets the picture how big the piece of the puzzle he received is.

**Pull request merged**